.github/workflows/lint.yaml

```diff
@@ -2,24 +2,50 @@ name: "Lint for Rails Projects"
 on:
   workflow_dispatch:
     inputs:
+      baseTarget:
+        description: "Used to enable the linter to run on the base image"
+        required: false
+        type: boolean
```

Comment:
I think there was a misunderstanding here. These should be the same as in build. That means they should be strings, not booleans and then below we should use the same `${{ inputs.baseTarget != '' }}` to check them. That way there is only one format across all the files for targets.